android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -153,7 +155,10 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public void onBackPressed(){
         if((mSessions.isEmpty())){
             noSessionsView.setVisibility(View.VISIBLE);
+        } else {
+            sessionRecyclerView.setVisibility(View.VISIBLE);
```

Comment:
Yes, but one is for noSessionsView and second is for sessionRecyclerView. 
On clicking we are setting visibility of both to GONE. So on pressing back button one of them should be visible